clients/src/main/resources/common/message/UpdateMetadataRequest.json

```diff
@@ -16,7 +16,7 @@
 {
   "apiKey": 6,
   "type": "request",
-  "listeners": ["zkBroker"],
+  "listeners": [],
```

Comment:
Do we want to remove this request, if it doesn't support any listeners.

Reply:
We do, but that's a bit involved. I have a JIRA related to this that I have assigned to myself.

Reply:
For clarity, you can't delete the json file, we need to update the generator to support requests that are no longer supported.

Reply:
#18497 includes the generator change I mentioned here.